crates/pop-parachains/src/new_parachain.rs

```diff
@@ -21,7 +21,7 @@ pub fn instantiate_template_dir(
 ) -> Result<Option<String>> {
 	sanitize(target)?;
 
-	if matches!(template, &Template::Base) {
+	if template.matches(&Provider::Pop) {
```

Comment:
This seems incorrect?

Reply:
No, for external parachains we only pull the repo from Github. For Pop ones we instantiate the parachain (customization flags, generate zombienet config...).
Before was hardcoded only for Base parachain, with this change will take all Pop templates